### DISCUSSION TITLE
A few ideas that would make OpenGlass a software for beginners and experts

### DISCUSSION THREAD

**Discussion by Milicent-Heeler** [ALTaleX531/OpenGlass#69] [Ideas]
First, I think there should be a settings menu. This will make it easier for beginners to tweak the settings. Next I think like Glass8, there should be a installer for various systems. Like for those whose running x64 or x86 or AMD64. There should also be a setting to get the aero theme/Aero Taskbar and start menu. Also there should be feature for advanced users were you can choose what will the caption buttons look like.

**Comment by ALTaleX531** [comment]
> First, I think there should be a settings menu. This will make it easier for beginners to tweak the settings. Next I think like Glass8, there should be a installer for various systems. Like for those whose running x64 or x86 or AMD64. 

Unfortunately, your request cannot be fulfilled. It doesn't make any sense to spend too much time on non-core implementations, I don't have that much time to devote to OpenGlass development.

> There should also be a setting to get the aero theme/Aero Taskbar and start menu. 

Here's the original clarification from glass8 website, I think these words also make sense for OpenGlass.
> Why is there no blur effect in the region/window/etc.?
Is it possible to enable the blur effect on the taskbar (Alt-Tab window, any other window etc.) ?
Remember that DWM is the rendering engine only and Aero Glass for Win8+ is the extension for it. It will apply glass effect only on the regions which are marked to have blur behind. It does not alter any region behaviour on its own. If some application wants to have glass effect somewhere, it must enable it explicitely. The taskbar belongs to Windows Explorer shell, thus you must enable blur behind on your own by calling DWM API function [DwmEnableBlurBehindWindow](http://msdn.microsoft.com/en-us/library/windows/desktop/aa969508(v=vs.85).aspx). Blur effect will also apply tint of glass color.

OpenGlass provides an additional feature called `GlassOverrideAccent`, which overrides any blur implemented using accent, this feature used to exist in glass8 as well, but has been removed since then.

> Also there should be feature for advanced users were you can choose what will the caption buttons look like.

OpenGlass does not include the ability to customize the caption buttons.
